app/controllers/camps_controller.rb

```diff
@@ -76,10 +75,21 @@ def update
   end
 
   def tag
-    @camp.update_attributes(tag_list: params.require(:camp).require(:tag_list))
+    @camp.update_attributes(tag_list: @camp.tag_list.add(params.require(:camp).require(:tag_list)))
+    respond_to do |format|
```

Comment:
`update_attributes` is an alias for the shorter, more common `update` method
Also, no need for a respond_to here I believe, you should be able to get away with
```
def tag
  @camp.update(tag_list: @camp.tag_list.add(tag_params))
  render json: @camp.tags
end

def tag_params
  params.require(:camp).require(:tag_list)
end
```

Reply:
Thanks!